server/app/views/applicant/NorthStarApplicantProgramBlockEditView.java

```diff
@@ -41,4 +43,19 @@ private String getFormAction(ApplicationBaseView.Params params) {
             ApplicantRequestedAction.NEXT_BLOCK)
         .url();
   }
+
+  private String getReviewUrl(ApplicationBaseView.Params params) {
+    return applicantRoutes.review(params.profile(), params.applicantId(), params.programId()).url();
+  }
```

Comment:
Quick drive-by here: I'm currently updating the "Review" and "Previous" buttons to also save the user's answers, which means they should also be submit buttons and not an `<a>` tags. See https://github.com/civiform/civiform/pull/6544 and https://github.com/civiform/civiform/pull/6547. Can we have the buttons in the north star also save answers?

Reply:
updated. good catch!